tfjs-backend-webgpu/src/unary_op_util.ts

```diff
@@ -114,6 +117,22 @@ const ELU_VEC4 = `
   }
   return resFloat;
 `;
+const ERF = `
+  // Error function is calculated approximately with elementary function.
+  // See "Handbook of Mathematical Functions with Formulas,
+  // Graphs, and Mathematical Tables", Abramowitz and Stegun.
+  let p = ${backend_util.ERF_P};
+  let a1 = ${backend_util.ERF_A1};
+  let a2 = ${backend_util.ERF_A2};
+  let a3 = ${backend_util.ERF_A3};
+  let a4 = ${backend_util.ERF_A4};
+  let a5 = ${backend_util.ERF_A5};
+
+  let sign = sign(a);
+  let absA = abs(a);
+  let t = 1.0 / (1.0 + p * absA);
+  return sign * (1.0 - (((((a5*t + a4)*t) + a3)*t + a2)*t + a1)*t*exp(-absA*absA));
```

Comment:
I know this is derived from WebGL code, but it's better to have space before and after operators.